data/consent-page.js

```diff
@@ -0,0 +1,66 @@
+const yo = require('yo-yo');
+
+// Build the study header
+const buildStudyTitle = studyName => {
+  const titleText = studyName || 'Participate in a new Firefox Study';
+  return yo`<h1>${titleText}</h1>`;
+};
+
+// Build the opt-in button
+const buildParticipateButton = (buttonText, buttonClasses) => {
+  const text = buttonText || `Try ${self.options.addonName}`;
+  return yo`
+  <button id="install-addon" class=${buttonClasses || 'primary'}>
```

Comment:
Nit: I think we should write this as

```
return yo`
  <button id="install-addon" class=${buttonClasses || 'primary'}>
    ${text}
  </button>
`;
```

That is, a line ending with a backtick should indent the next line, and should be dedented at the end. I think that lines up better with the rest of our styling.